en_US.json

```diff
@@ -252,6 +252,11 @@
 	"certificate.dns01challenge.opt.desc": "This option allows us to automatically create a certificate supporting each of your hostnames after verifying your ownership of them.  To verify, you must modify your DNS server to add a special DNS TXT record for each hostname. Once the process is completed, you can remove those TXT records.",
 	"certificate.dns01challenge.radio.title": "DNS-01 challenge",
 	"certificate.dns01challenge.requesting": "Requesting",
+	"certificate.dns01challenge.steps.s1.title": "Initial verification",
```

Comment:
The initial letter of verification needs to be capitalized to be consistent with other titles here below.